src/api.tsx

```diff
@@ -0,0 +1,74 @@
+const BASE_URL = 'https://mate.academy/students-ap';
```

Comment:
```suggestion
const BASE_URL = 'https://mate.academy/students-api';
```

Reply:
<img width="1411" alt="image" src="https://github.com/mate-academy/react_todo-app/assets/98764641/2e9b5f71-a606-4c1a-878d-0108fe21aa34">

Reply:
please fix it because I can't check how it works

Reply:
Done, I am sorry, I have made it incorrect to check error message.